decred/dbg-pudb.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Can we name this file something more memorable?

Reply:
The name is meant to allow typing less than six chars, hopefully as little as three, before being able to tab for autocompletion. :slightly_smiling_face: It'd be nice if any more memorable name still allowed that.

Reply:
Agreed in chat to change it to `debugger.sh`.